### DISCUSSION TITLE
Python 3.12 Support

### DISCUSSION THREAD

**Discussion by RenatoFarruggio** [great-expectations/great_expectations#10462] [General]
Hello,

In the README it says that Great Expectations is officially supported up to Python 3.11, with Python 3.12 being marked as experimental. I’m currently using Python 3.12 and would like to integrate Great Expectations into my projects in the future.

Could anyone provide more details on the current status of Python 3.12 support? Are there known issues or dependencies blocking full support? Additionally, is there a roadmap or timeline for when Python 3.12 compatibility will be fully released?

Thanks in advance!

Best regards,
Renato

**Comment by Kilo59** [comment]
@RenatoFarruggio This was officially added in version `1.1.3`
https://github.com/great-expectations/great_expectations/releases/tag/1.1.3 

- https://github.com/great-expectations/great_expectations/pull/10503

**Reply by RenatoFarruggio** [reply]
Perfect, thank you!
